recipe/meta.yaml

```diff
@@ -23,7 +20,8 @@ requirements:
   host:
     - python
     - pip
-    - numpy
+    - cython
+    - {{ pin_compatible('numpy') }}
```

Comment:
this should be `numpy` imho

Reply:
Thanks @bgruening! Addressed in https://github.com/conda-forge/biom-format-feedstock/pull/6/commits/a5fa5a662e11998421a095ff774eb02a00cef265.